AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Requests/SendRequestStatusToGetASmokeAlarmHandlerShould.cs

```diff
@@ -0,0 +1,113 @@
+
+    using Moq;
+
+    public class SendRequestStatusToGetASmokeAlarmHandlerShould : InMemoryContextTest
+    {
+        private readonly Guid requestId;
+
+        public SendRequestStatusToGetASmokeAlarmHandlerShould()
+        {
+            this.requestId = Guid.NewGuid();
+            this.Context.Requests.Add(new Request
+            {
+                RequestId = this.requestId,
+                Source = RequestSource.Api
+            });
+            this.Context.SaveChanges();
+        }
+
+        [Fact]
+        public async Task NotQueueUpAnythingIfTheRequestStatusIsNotASupportedStatus()
+        {
+            var notification = new RequestStatusChangedNotification
+            {
+                RequestId = this.requestId,
+                NewStatus = RequestStatus.PendingConfirmation
+            };
+
+            var backgroundJobClient = new Mock<IBackgroundJobClient>();
+            var sut = new SendRequestStatusToGetASmokeAlarmHandler(this.Context, backgroundJobClient.Object);
+
+            await sut.Handle(notification);
+
+            backgroundJobClient.Verify(x => x.Create(It.IsAny<Job>(), It.IsAny<EnqueuedState>()), Times.Never);
+        }
+
+        [Fact]
+        public async Task NotQueueAnythingIfTheRequestCannotBeFound()
+        {
+            var notification = new RequestStatusChangedNotification
+            {
+                RequestId = Guid.NewGuid(),
+                NewStatus = RequestStatus.Completed
+            };
+
+            var backgroundJobClient = new Mock<IBackgroundJobClient>();
+            var sut = new SendRequestStatusToGetASmokeAlarmHandler(this.Context, backgroundJobClient.Object);
+
+            await sut.Handle(notification);
+
+            backgroundJobClient.Verify(x => x.Create(It.IsAny<Job>(), It.IsAny<EnqueuedState>()), Times.Never);
+        }
+
+        [Theory]
+        [InlineData(RequestStatus.Completed)]
+        [InlineData(RequestStatus.Canceled)]
+        [InlineData(RequestStatus.Assigned)]
+        [InlineData(RequestStatus.Unassigned)]
+        public async Task EnqueueAJobIfTheNewStatusIsASupportedStatus(RequestStatus newStatus)
+        {
+            var notification = new RequestStatusChangedNotification
+            {
+                RequestId = this.requestId,
+                NewStatus = newStatus
+            };
+
+            var backgroundJobClient = new Mock<IBackgroundJobClient>();
+            var sut = new SendRequestStatusToGetASmokeAlarmHandler(this.Context, backgroundJobClient.Object);
+
+            await sut.Handle(notification);
+
+            backgroundJobClient.Verify(x => x.Create(It.IsAny<Job>(), It.IsAny<EnqueuedState>()), Times.Once);
+        }
+
+        [Theory]
+        [InlineData(RequestStatus.Completed, "", true)]
+        [InlineData(RequestStatus.Canceled, "canceled", true)]
+        [InlineData(RequestStatus.Assigned, "in progress", true)]
+        [InlineData(RequestStatus.Unassigned, "new", false)]
```

Comment:
for the GASA value for the inline statuses, could you please change these to use the GasaStatus static string values instead of the string itself? 

example, instead of:
`[InlineData(RequestStatus.Canceled, "canceled", true)]`
do
`[InlineData(RequestStatus.Canceled, GasaStatus.Canceled, true)]`

this can be accomplished by changing the static string field values under GasaStatus from `static` to `const`.

Thanks